test/index.js

```diff
@@ -243,7 +248,12 @@ describe('hapi-setup Plugin', function () {
           },
           attributes: {
             name: 'bar',
-            version: '1.0.0'
+            version: '1.0.0',
+            pkg: {
+              version: '0.0.0'
```

Comment:
I'm assuming you're referring to the issue with 
```
pkg: {
    version: '0.0.0'
}
```

That's actually what is returned via Hapi in this case.  I wasn't adding it to not match.  So perhaps this is a symptom of a upstream problem.